packages/app/package.json

```diff
@@ -32,10 +32,10 @@
     "react-devtools": "^3.4.2",
     "react-test-renderer": "16.5.2",
     "relay-compiler": "1.6.2",
-    "relay-devtools": "^1.4.0"
-  },
-  "jest": {
-    "preset": "react-native"
+    "relay-devtools": "^1.4.0",
+    "graphql-query-test-mock": "^0.9.0",
+    "nock": "^10.0.1",
```

Comment:
why do we need this?

Reply:
@sibelius

Reply:
this is needed for graphql-query-test-mock

we should migrate to https://github.com/1stdibs/relay-mock-network-layer soon, it sounds more scalable and easier

Reply:
nice